pkg/executables/flux.go

```diff
@@ -39,7 +39,7 @@ func (f *Flux) BootstrapToolkitsComponentsGithub(ctx context.Context, cluster *t
 		"--repository", c.Github.Repository,
 		"--owner", c.Github.Owner,
 		"--path", c.ClusterConfigPath,
-		"--ssh-key-algorithm", privateKeyAlgorithm,
+		"--ssh-key-algorithm", defaultPrivateKeyAlgorithm,
```

Comment:
so for bootstrap github we always use "ecdsa", and for bootstrap git we let user choose ssh algo?

Reply:
based on documentation here https://fluxcd.io/docs/cmd/flux_bootstrap_github/
both github and git support all 3 algorithms

Reply:
https://github.com/aws/eks-anywhere/issues/2037